api/checks/api_test.go

```diff
@@ -0,0 +1,40 @@
+//+build small
```

Comment:
This should be a medium test because of the use of dev_appserver. And the file needs to be renamed.

Reply:
Done.